modules/src/main/java/org/jboss/additional/testsuite/jdkall/present/core/jca/DatasourcesActiveCountTestCase.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2013, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.jboss.additional.testsuite.jdkall.present.core.jca;
+
+import java.io.IOException;
+import org.jboss.eap.additional.testsuite.annotations.EapAdditionalTestsuite;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.as.test.integration.management.base.AbstractCliTestBase;
+import org.jboss.as.controller.descriptions.ModelDescriptionConstants;
+import org.jboss.as.test.integration.management.util.CLIOpResult;
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.ModelType;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Jiri Bilek
+ * jbilek@redhat.com on 12/03/18.
+ * Test for JBEAP-14635 JBJCA-1375
+ *
+ * ActiveCount of data-source is not correct after reloaded
+ * ActiveCount in data-source statistics is not correct after executing reload CLI command.
+ *
+ * Sometimes PoolStatisticsImpl is cleared too late during AS reload, and some
+ * connections are already counted. That results in wrong stats count (e.g. "ActiveCount").
+ * The test trying find out ActiveCount 'COUNT_OF_RETRYING' times
+ *
+ */
+@RunWith(Arquillian.class)
+@RunAsClient
+//@EapAdditionalTestsuite({"modules/testcases/jdkAll/Eap71x-Proposed/management/src/main/java#7.1.4","modules/testcases/jdkAll/Eap71x/management/src/main/java#7.1.4","modules/testcases/jdkAll/Wildfly/management/src/main/java#14.0.0"})
```

Comment:
Could we change //@ EapAdditionalTestsuite to //@ AdditionalTestsuite ? @ EapAdditionalTestsuite distributes the code also when it is commented. This is done in order to be used with any programming language, configurations, and other .txt, .xml, etc resources.